moveit_py/src/moveit/moveit_core/robot_trajectory/robot_trajectory.cpp

```diff
@@ -134,6 +164,19 @@ void init_robot_trajectory(py::module& m)
            Returns:
                list of float: The duration from previous of each waypoint in the trajectory.
            )")
+      .def("retime_trajectory", &moveit_py::bind_robot_trajectory::retimeTrajectory, py::arg("algorithm"),
```

Comment:
why would you call a method on a class `RobotTrajectory` `retime_trajectory` instead of simply `retime`?
In moveit, the reason for the full name is that it's a member of the `MoveGroupInterface` and not the trajectory.